Fw/Types/CMakeLists.txt

```diff
@@ -18,9 +18,10 @@ set(SOURCE_FILES
   "${CMAKE_CURRENT_LIST_DIR}/StringUtils.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/StringToNumber.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/Types.fpp"
+  "${CMAKE_CURRENT_LIST_DIR}/snprintf_format.cpp"
 )
 set(MOD_DEPS
-  Fw/Cfg
+  __fprime_config
```

Comment:
Since I cannot do a generator on a global property, I have to use a hidden target name